stock_lot_multi_image/readme/CONTEXT.md

```diff
@@ -0,0 +1,3 @@
+Sometimes it is necessary to have photos of a particular stock lot (item) in addition to the product-level photos (for product.template or product.variant). 
```

Comment:
Sometimes it is necessary to have pictures of a specific product that is tracked by serial numbers.
For example you manage a stock of used laptops where each model is a product variant and each laptop is an item with a unique serial number. And you would like to have photos of each specific laptop, not just the pictures of the model from the official website.  
However by default in Odoo you can add pictures only to a product template or a product variant.